src/components/AppLayout/Header/components/NetworkSelector.tsx

```diff
@@ -1,5 +1,4 @@
-import React, { ReactElement, useRef, Fragment, useCallback } from 'react'
-import { useDispatch } from 'react-redux'
+import React, { ReactElement, useRef, Fragment } from 'react'
```

Comment:
Please pull from dev, React now shouldn't be imported.

Reply:
I have added the rule to eslint now and removed all instances of `react` imports